docs/setting_up_your_development_environment.md

```diff
@@ -110,8 +127,8 @@ annotate the test page. If it asks for a username and password you can use
 
 ### Via
 
-[Via](https://github.com/hypothesis/viahtml) is a proxy that injects the client
-into arbitrary PDFs. To install and run it:
+[Via](https://github.com/hypothesis/via) is a proxy that injects the Hypothesis client
+into PDFs so users can annotate them. To install and run it:
```

Comment:
The GitHub repo link was wrong. Also re-worded to clarify what the point of Via is

Reply:
Technically, Via serves a PDF viewer with Hypothesis enabled, rather than modifying the PDF itself.